lib/miq_automation_engine/service_models/miq_ae_service_physical_server.rb

```diff
@@ -0,0 +1,11 @@
+module MiqAeMethodService
+  class MiqAeServicePhysicalServer < MiqAeServiceModelBase
+    expose :ext_management_system, :association => true
+    expose :physical_servers,      :association => true
```

Comment:
Does not make any sense, I committed a mistake.Does not make any sense, I committed a mistake.